world/templates/template_manager.py

```diff
@@ -8,8 +8,8 @@
 
 class TemplateManager(models.Manager):
     def accessible_by(self, char):
-        return self.get_queryset().filter(Q(owner=char.roster.current_account) | Q(templategrantee__grantee=char.roster) | Q(access_level="OP"))
+        return self.get_queryset().filter(Q(owner=char.roster.current_account) | Q(templategrantee__grantee=char.roster) | Q(access_level="OP")).distinct()
```

Comment:
Yeah, I always have trouble remembering when not having `.distinct()` will produce duplicates, but usually happens with complex queries in which you could have multiple matches or when having joins that could also produce multiple matches.